From 665e3d2e94224c7ae394158a0a5992a72efa4221 Mon Sep 17 00:00:00 2001
From: Roman Yurchak <rth.yurchak@gmail.com>
Date: Wed, 6 Apr 2022 21:19:55 -0700
Subject: [PATCH 05/17] disable blas detection

BLAS and LAPACK aren't available on host because we only cross compile these
libraries (see CLAPACK/meta.yaml). Scipy tries to detect these libraries and
if it fails to find them errors out the build. But we have them installed
correctly for our target, so we just disable this detection mechanism.
---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index e4ba8f669..a1a7c4e25 100755
--- a/setup.py
+++ b/setup.py
@@ -415,7 +415,7 @@ def configuration(parent_package='', top_path=None):
 
     lapack_opt = get_info('lapack_opt')
 
-    if not lapack_opt:
+    if not lapack_opt and False:
         if sys.platform == "darwin":
             msg = ('No BLAS/LAPACK libraries found. '
                    'Note: Accelerate is no longer supported.')
-- 
2.25.1

